lte/gateway/python/integ_tests/common/subscriber_db_client.py

```diff
@@ -18,14 +18,17 @@
 import time
 
 import grpc
-from integ_tests.gateway.rpc import get_gateway_hw_id, get_rpc_channel
+from integ_tests.gateway.rpc import get_gateway_hw_id, get_rpc_channel, get_hss_rpc_channel
```

Comment:
There was a Python format check complains about order of import statements, please follow as it suggested.